data-config.json

```diff
@@ -17,6 +17,7 @@
         "css": true,
         "csstxt": true,
         "html": true,
+        "html-content": true,
```

Comment:
Please toggle this to `false` in the demo-config file since this is a new feature that some people may not necessarily want.